### PR TITLE
Mechanical Curtains

### PR DESCRIPTION
**Pull request by AlinaStarkova**
## About The Pull Request

adds a new variant of fancy curtains that are button operated only

## Why It's Good For The Game

good for better mapping, I have seen some mappers wish to utilize fancy curtains on their rooms windows (for example, a psychologist office) but wanted to be able to have privacy when with a patient, which curtains can be opened from either side of the window, and shutters just dont fit the theme, this way mappers can add in curtains which are button operated and cant be opened manually, basically, it functions like a shutter, only it looks and acts like curtains (can be destroyed like one, etc... it is literally a curtain subtype)

## Changelog
:cl: Nari Harimoto
add: Mechanical Curtains, a button only operated variant of Fancy Curtains
/:cl:

### CODE REVIEW

**Comment by Faitorius** [comment]
First

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -708,3 +708,37 @@
 /obj/structure/curtain/cloth/fancy
 	icon_type = "cur_fancy"
 	icon_state = "cur_fancy-open"
+
+/obj/structure/curtain/cloth/fancy/mechanical
+	var/id = null
+	icon_type = "cur_fancy"
+	icon_state = "cur_fancy-open"
```
Comment (Fikou) [inline code/game/objects/structures/watercloset.dm:715]: these are inherited from parent, either remove them or make this not a subtype

Comment (AlinaStarkova) [reply code/game/objects/structures/watercloset.dm:715]: ok, at work right now, so it will be awhile before i fix that

**Comment by KathrinBailey** [comment]
I was gonna do this, nice, glad someone beat me to it.
:eyes:

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-3). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**